### PR TITLE
AddLinkFix

### PR DESCRIPTION
**Pull request by Lunalo**
@lilyclements 

You can test this one now 
just a small fix

### CODE REVIEW

**Comment by Lunalo** [comment]
fixes #3211

**Comment by lilyclements** [comment]
Can you add in for OK to be disabled if the background colour is red/if the link cannot be made

File: instat/dlgAddLink.vb [hunk 1]
```diff
@@ -119,9 +120,11 @@ Public Class dlgAddLink
                 clsColumnNames.AddParameter("to_columns", "c(" & strKeyColumns & ")")
                 bCanAutoLink = chrKeyColumns.ToArray.All(Function(strCol) strColumnNames.Contains(strCol))
                 If bCanAutoLink Then
-                    lviTemp.ForeColor = Color.Green
+                    lviTemp.BackColor = Color.Green
+                    ucrBase.OKEnabled(True)
```
Comment (lilyclements) [inline instat/dlgAddLink.vb]: This should be LightGreen not Green, like in the keys dialog
Additionally, it should be LightCoral not Red.

Comment (dannyparsons) [inline instat/dlgAddLink.vb]: @Lunalo it's not Lily's responsibility to tell you what to do! Couldn't you have checked this on the Add Key dialog yourself when you were doing this?

Comment (Lunalo) [inline instat/dlgAddLink.vb]: @lilyclements 

That is fixed.

thanks

**Comment by lilyclements** [comment]
@dannyparsons this is ready to merge

**Pull request merged**